test/passing/tests/extensions-indent.ml.ref

```diff
@@ -198,7 +198,7 @@ let () =
   [%(* 0 *)
     (* 0.5 *)
        test
-       () (* 1 *) [@foo (* 2 *) "bar"] (* 3 *)]
+       begin end (* 1 *) [@foo (* 2 *) "bar"] (* 3 *)]
```

Comment:
Is this intended?

Reply:
It is less incorrect than the current formatting, since there is begin/end in the original piece of code